grails-app/views/stockMovement/show.gsp

```diff
@@ -370,10 +370,10 @@
                             </td>
                         </tr>
 
-                        <g:if test="${stockMovement.requisition.status in [RequisitionStatus.APPROVED, RequisitionStatus.REJECTED]}">
-                            <g:set var="person" value="${stockMovement.requisition.approvedBy ?: stockMovement.requisition.rejectedBy}" />
-                            <g:set var="date" value="${stockMovement.requisition.dateApproved ?: stockMovement.requisition.dateRejected}" />
-                            <g:set var="label" value="${stockMovement.requisition.status == RequisitionStatus.APPROVED ? 'dateApproved' : 'dateRejected'}" />
+                        <g:if test="${stockMovement?.requisition?.status in [RequisitionStatus.APPROVED, RequisitionStatus.REJECTED]}">
+                            <g:set var="person" value="${stockMovement?.requisition?.approvedBy ?: stockMovement?.requisition?.rejectedBy}" />
+                            <g:set var="date" value="${stockMovement?.requisition?.dateApproved ?: stockMovement?.requisition?.dateRejected}" />
+                            <g:set var="label" value="${stockMovement?.requisition?.status == RequisitionStatus.APPROVED ? 'dateApproved' : 'dateRejected'}" />
```

Comment:
it doesn't matter at this point, but just wanted to mention that the fact, you've gone into that if means, that there is `stockMovement.requisition`, so you wouldn't have to add the null safe operators inside the if.